nat-lab/tests/test_direct_connection.py

```diff
@@ -499,7 +490,7 @@ async def test_direct_short_connection_loss(
                     await asyncio.wait_for(task, 1)
 
         async with Ping(alpha_connection, beta.ip_addresses[0]).run() as ping:
-            await testing.wait_lengthy(ping.wait_for_next_ping())
+            await ping.wait_for_next_ping(60)
```

Comment:
These as well 🙈

Reply:
You see, these can happen randomly everywhere, can't be putting all of these to CONSTANTS :smile: